lib/buildenv/default.nix

```diff
@@ -0,0 +1,84 @@
+# buildEnv creates a tree of symlinks to the specified paths.  This is
+# a fork of the buildEnv in the Nix distribution.  Most changes should
+# eventually be merged back into the Nix distribution.
+
+{ buildPackages, runCommand, lib, substituteAll }:
```

Comment:
This breaks evaluation of `lib`:
```
> flox eval --json .#lib
error: evaluation aborted with the following error message: 'Function called without required argument "buildPackages" at /nix/store/74xcjbzkljbs4avnqi2vq85mf46gnyb3-source/lib/buildenv/default.nix:5'
```

buildEnv is a bit weird because I think it needs to be system specific (it depends on buildPackages), but I think capacitor doesn't make things in lib system specific. You can move buildenv from lib to pkgs, but then lib/mkEnv can't access it. We might be able to thread it all the way through from the floxEnvs plugin but that would be kind of a pain. We could also move mkEnv to pkgs. I dunno the best path forward, probably a question for @ysndr

Reply:
Ditto above

Reply:
Yannik's suggestions to change the way buildenv was called fixed this - now it succeeds (slightly modified) as follows:
```
flox [flox/experimental default] brantley@Michaels-MacBook-Pro flox % flox eval --json .#lib --apply 'x: builtins.deepSeq x null'
null
```